README.adoc

```diff
@@ -541,257 +541,11 @@ List<File> asciidocFiles = directoryWalker.scan(); // <2>
 Instead of converting an AsciiDoc document, you may want to parse the document to read information it contains or navigate the document's structure.
 AsciidoctorJ let's you do this!
 
-There are two approaches you can take to read the structure of an AsciiDoc document with AsciidoctorJ.
+AsciidoctorJ provides a model of the abstract syntax tree of the document.
+These object are directly linked to the internal representation.
```

Comment:
should be "objects" given there is plural

Reply:
Thanks for seeing this! :)